.ci/aarch64/install_rom_aarch64.sh

```diff
@@ -78,7 +78,7 @@ prepare_default_uefi()
 	else
 		local efi_url="https://releases.linaro.org/components/kernel/uefi-linaro/latest/release/qemu64/QEMU_EFI.fd"
 		sudo mkdir -p "${EFI_DEFAULT_DIR}"
-		sudo curl -LO "${EFI_DEFAULT_DIR}/$efi_url"
+		sudo curl -L $efi_url -o "${EFI_DEFAULT_PATH}"
```

Comment:
I'm not clear why this part of the change is required as it seems to do the same thing?

Reply:
I tried on my local machine and found that two kinds of cmd executed as below:
```sh
[root@img-create aarch64]# curl -LO /usr/share/qemu-efi-aarch64/https://releases.linaro.org/components/kernel/uefi-linaro/latest/release/qemu64/QEMU_EFI.fd
curl: (3) <url> malformed
[root@img-create aarch64]# curl -L https://releases.linaro.org/components/kernel/uefi-linaro/latest/release/qemu64/QEMU_EFI.fd -o /usr/share/qemu-efi-aarch64/QEMU_EFI.fd
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
100 2048k  100 2048k    0     0   345k      0  0:00:05  0:00:05 --:--:--  529k
[root@img-create aarch64]# curl -LO /usr/share/qemu-efi-aarch64/ https://releases.linaro.org/components/kernel/uefi-linaro/latest/release/qemu64/QEMU_EFI.fd
curl: Remote file name has no length!
curl: try 'curl --help' or 'curl --manual' for more information
^C
[root@img-create aarch64]# curl -V
curl 7.29.0 (aarch64-redhat-linux-gnu) libcurl/7.29.0 NSS/3.28.4 zlib/1.2.7 libidn/1.28 libssh2/1.4.3
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp scp sftp smtp smtps telnet tftp
Features: AsynchDNS GSS-Negotiate IDN IPv6 Largefile NTLM NTLM_WB SSL libz unix-sockets
```
So I think the `curl -LO` may have been broken here.

Reply:
With `-O` you can only pass the URL so do this work for you?:

```bash
curl -LO https://releases.linaro.org/components/kernel/uefi-linaro/latest/release/qemu64/QEMU_EFI.fd
```

But also it's not ideal to run `curl` with `sudo` from a security perspective. Ideally, you'd do something like this (but using variables as approprate):

```
tmpdir=$(mktemp -d)
pushd "$tmpdir" || die "..."
curl -LO https://releases.linaro.org/components/kernel/uefi-linaro/latest/release/qemu64/QEMU_EFI.fd
sudo install -o root -g root -m 640 QEMU_EFI.fd /usr/share/qemu-efi-aarch64/
popd
rm -rf "$tmpdir"
```

Reply:
Hello @Bevisy -, thanks for reporting this. 
Yeah, I think QEMU_EFI.fd can be put under kata home path like you does. Considering @jodh-intel's comments, what about using curl without "sudo" if it works for you.

Reply:
OK, the fix has been redone and verified on my local machine.